README.md

```diff
@@ -1,24 +1,125 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-* System dependencies
+- [📝 License](#license)
 
-* Configuration
+# 📖 Blog App <a name="about-blog-project"></a>
 
-* Database creation
+This is a blog application that shows the list of posts and enables readers to interact with them by adding comments and liking posts
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+- <a href="https://www.ruby-lang.org/">Ruby</a>
+- <a href="https://rubyonrails.org/">Rails</a>
+- <a href="https://www.postgresql.org/">PostgreSQL</a>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- Create posts.
+- List all the posts.
+- Add comments and likes to a post.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+To run this project you need the following dependencies:
+
+- Ruby: `3.2.2`
+- Rails: `7.0.4.3+`
+- PostgreSQL: `14.7+`
+
+### Setup
+
+- Clone this repository to your local machine:
+
+```sh
+  git clone https://github.com/chepkok3/Blog_App.git
+  cd Blog_App
+```
+
+- Install dependencies:
+
+```sh
+  bundle install
+```
+
+- Recreate Data Base:
+
+```sh
+  rake db:schema:load
+```
+
+### Usage
+
+To run the project, execute the following command in the terminal:
+
+```sh
+  cd Blog_App
+  rails s
+```
+
+## 👥 Author <a name="author"></a>
+
+👤 **Stanley Kibor**
+
+- GitHub: [@chepkok3](https://github.com/chepkok3)
+- Twitter: [@chepkok3](https://twitter.com/home)
+- LinkedIn: [@chepkok3](https://www.linkedin.com/in/kibor-stanley-350b8a123/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add validations to the model.
+- Create controllers.
+- Create a web interface.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/chepkok3/Blog_App/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Much appreciation to the Microverse community for providing the guidance and support needed to complete this project:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The MIT link is broken, when I click on it I get a `page not found`. Please kindly fix the broken link 😉 🙃